Src/WitsmlExplorer.Frontend/components/TopRightCornerMenu.tsx

```diff
@@ -74,6 +87,13 @@ const TopRightCornerMenu = (): React.ReactElement => {
 
   return (
     <Layout>
+      { currentWitsmlLoginState.username && 
+      <Button variant="ghost">
+        <Icon name="person" color={colors.text.staticIconsTertiary}/>
+        { currentWitsmlLoginState.username } 
+      </Button> 
+      }
+      <ManageServerButton />
```

Comment:
This button is styled differently from all the others resulting in inconsistent UI.